services/crm/src/main/java/com/gym/crm/service/mq/MessageProducer.java

```diff
@@ -0,0 +1,41 @@
+package com.gym.crm.service.mq;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule;
+import com.gym.crm.model.dto.request.RegisterRequest;
+import com.gym.crm.model.dto.request.TrainerWorkload;
+import jakarta.jms.Queue;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.activemq.command.ActiveMQQueue;
+import org.springframework.jms.core.JmsTemplate;
+import org.springframework.stereotype.Service;
+
+@Service
+@RequiredArgsConstructor
+@Slf4j
+public class MessageProducer {
+
+    private final JmsTemplate jmsTemplate;
+    private final Queue workloadQueue = new ActiveMQQueue("workload.queue");
```

Comment:
Please not use magichttps://stoic.software/articles/magic-strings-the-anti-pattern/ string